### PR TITLE
Rewrap azure node stop and start methods

### PR DESCRIPTION
**Pull request by suchita-g**
Signed-off-by: Suchita Gatfane <sgatfane@redhat.com>

### CODE REVIEW

**Comment by suchita-g** [comment]
+1 Verified 
Reference test execution https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/15909/

File: ocs_ci/utility/azure_utils.py [hunk 1]
```diff
@@ -315,28 +317,48 @@ def get_disk_state(self, volume_name):
...
+        for node_name in node_names:
+            result = self.compute_client.virtual_machines.start(
+                self.cluster_resource_group, node_name
+            )
+            result.wait()
+
+        if wait:
+            # When the node is reachable then the node reaches status Ready.
+            logger.info(f"Waiting for nodes: {node_names} to reach ready state")
+            wait_for_nodes_status(
```
Comment (ebenahar) [inline ocs_ci/utility/azure_utils.py:340]: Node related operations should not be done in this file. This file should be for Azure resources related operations and nodes is an Openshift resource. Please follow the same convention we have in other platforms' low level modules:
https://github.com/red-hat-storage/ocs-ci/blob/a6902fa00f24d5b5fc4bc0939a1f2249e537e57d/ocs_ci/utility/aws.py#L471
https://github.com/red-hat-storage/ocs-ci/blob/a6902fa00f24d5b5fc4bc0939a1f2249e537e57d/ocs_ci/utility/vsphere.py#L442

Comment (suchita-g) [reply ocs_ci/utility/azure_utils.py:340]: Yes, this code wrap is not appropriate. I am closing this PR

**Comment by suchita-g** [comment]
Yes, this code wrap is not appropriate. I am closing this PR.

**Pull request closed without merging**